foundations/installations/setting_up_git.md

```diff
@@ -99,7 +99,17 @@ You will need to install Git from the source by following the instructions from
 
 ### Step 2: Configure Git and GitHub
 
-#### Step 2.1: Setup Git
+#### Step 2.1: Create a GitHub Account
+
+Go to [GitHub.com](https://github.com/) and create an account! During the account setup, it will ask you for an email address. This needs to be a real email, and will be used by default to identify your contributions. If you are privacy concious, or just don't want your email address to be publically available, make sure you tick the following two boxes on the [Email Settings](https://github.com/settings/emails) page after you have signed in:
```

Comment:
```suggestion
Go to [GitHub.com](https://github.com/) and create an account! During the account setup, it will ask you for an email address. This needs to be a real email, and will be used by default to identify your contributions. If you are privacy conscious, or just don't want your email address to be publicly available, make sure you tick the following two boxes on the [Email Settings](https://github.com/settings/emails) page after you have signed in:
```